test/browser_tests/spec_suites/templates/browse-filterable.js

```diff
@@ -1,60 +1,61 @@
 'use strict';
 
-var Blog = require(
-    '../../page_objects/page_blog.js'
-  );
+var Blog = require( '../../page_objects/page_blog.js' );
 
-describe( 'Pagination', function() {
-  var page;
+describe( 'Browse filterable', function() {
 
-  beforeEach( function() {
-    page = new Blog();
-    page.get();
-  } );
+  describe( 'pagination', function() {
+    var page;
 
-  it( 'should navigate to the first page of filtered results', function() {
-    page.searchFilterBtn.click();
-    browser.sleep( 1000 );
+    beforeEach( function() {
+      page = new Blog();
+      page.get();
+    } );
 
-    page.searchCategoryLabel.click();
-    page.searchFilterSubmitBtn.click();
-    browser.sleep( 1000 );
+    it( 'should navigate to the first page of filtered results', function() {
+      page.searchFilterBtn.click();
+      browser.sleep( 1000 );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2267/commits/423f89bf512f9217e2c473de487b2acddcd37ca6